drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/test/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/strategies/BusinessCentralDMODataManagementStrategyTest.java

```diff
@@ -81,7 +87,7 @@ public void setup() {
         factModelTupleLocal = new FactModelTuple(new TreeMap<>(), new TreeMap<>());
         factModelTreeHolderlocal = new BusinessCentralDMODataManagementStrategy.ResultHolder();
         factModelTreeHolderlocal.setFactModelTuple(factModelTupleLocal);
-        when(oracleMock.getFQCNByFactName(TestProperties.FACT_NAME)).thenReturn(TestProperties.FULL_FACT_CLASSNAME);
+        when(oracleMock.getFQCNByFactName(TestProperties.FACT_NAME)).thenReturn(TestProperties.FULL_CLASS_NAME);
```

Comment:
@danielezonca  The behavior is not changed, I applied this change because `FULL_CLASS_NAME` is a more realistic output for that method.
Please consider:
`FULL_CLASS_NAME = test.scesim.TestClass` 
`FULL_FACT_CLASSNAME = FULL_FACT_CLASSNAME`
Method `getFactModel` extract the className and the package, as you mentioned in next comment, I can check the the package is correctly extracted, and eventually have a more complete test.